results/17-LV-GenomicVariantTest.Rmd

```diff
@@ -0,0 +1,146 @@
+---
+title: "17 - LV Genomic Variant Test"
+author: "Sara Gosline"
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    code_folding: hide
+    toc: true
+    toc_float: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(synapser)
+synLogin()
+library(tidyverse)
+library(feather)
+```
+
+# Get Data
+
+This is nearly entirely based on the code in notebook 09 and that in 11. 
+
+We have latent variable expression analysis data
+- [Latent Variable Feather File](https://www.synapse.org/#!Synapse:syn21046991/tables/)
+
+For this data we are also using any data for which there are gene variants (cNFs, pNFs, MPNSTs):
+- [Exome-Seq variants](https://www.synapse.org/#!Synapse:syn20554939/tables/)
+- [WGS Variants](https://www.synapse.org/#!Synapse:syn20551862/tables/) 
+
+Let's see if there are any LVs that split based on gene variant. Because we're having trouble scaling with the number of latent variables, I only look at variants that occur in less than 5% of the population. notice this is a difference from notebook #11.
+
+```{r,warning=FALSE}
+wgs.vars=synTableQuery("SELECT Hugo_Symbol,Protein_position,specimenID,IMPACT,FILTER,ExAC_AF FROM syn20551862")$asDataFrame()
```

Comment:
It seems gnomAD scores have more coverage than ExAC_AF (https://www.genomeweb.com/sequencing/gnomad-resource-introduced-ashg-meeting-doubles-exac-dataset). We should probably switch to using using gnomAD as our metric.

Reply:
ok, i still used a threshold of <0.05, does this make sense?

Reply:
I have used < 0.01 (albeit ExAC) in the earlier analyses given that some papers used that as their filtering threshold for common variants... 0.05 as threshold should work too if we can cite why we used that threshold.